<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgNewImage</class>
 <widget class="QWidget" name="WdgNewImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>txtName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtName">
       <property name="text">
        <string>untitled-1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="grpImage">
       <property name="title">
        <string>Image Size</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <spacer name="imageGroupSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout">
          <item>
           <layout class="QVBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblWidth">
              <property name="text">
               <string>&amp;Width:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>doubleWidth</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblHeight">
              <property name="text">
               <string>&amp;Height:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>doubleHeight</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="doubleWidth">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleHeight">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="cmbWidthUnit"/>
            </item>
            <item>
             <widget class="QComboBox" name="cmbHeightUnit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QToolButton" name="bnPortrait">
            <property name="toolTip">
             <string>Portrait</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="bnLandscape">
            <property name="toolTip">
             <string>Landscape</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblResolution">
            <property name="text">
             <string>Resolution:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleResolution">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>pixels-per-inch</string>
            </property>
            <property name="text">
             <string>ppi</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>191</width>
              <height>61</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="bnScreenSize">
            <property name="toolTip">
             <string>Set Width and Height to screen resolution.</string>
            </property>
            <property name="text">
             <string>&amp;Screen size</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Save Predefined Size as:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPredefinedName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bnSaveAsPredefined">
            <property name="toolTip">
             <string>Save the current dimensions</string>
            </property>
            <property name="text">
             <string>&amp;Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Predefined:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>cmbPredefined</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbPredefined">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpClipboard">
       <property name="title">
        <string>Clipboard</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="clipPreview">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="interactive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpMode">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="KisColorSpaceSelector" name="colorSpaceSelector" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpContents">
     <property name="title">
      <string>Contents</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblColor">
        <property name="text">
         <string>C&amp;anvas color:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>cmbColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorButton" name="cmbColor">
        <property name="color">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
        <property name="defaultColor">
         <color>
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Number of layers:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="KIntNumInput" name="intNumLayers">
        <property name="value">
         <number>2</number>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblOpacity">
        <property name="text">
         <string>&amp;Opacity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>sliderOpacity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="opacityPanel">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>Transparent</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderOpacity">
           <property name="toolTip">
            <string>Opacity of the background color</string>
           </property>
           <property name="whatsThis">
            <string>Opacity of the background color.</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickInterval">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>Opaque</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblDescription">
        <property name="text">
         <string>&amp;Description:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="buddy">
         <cstring>txtDescription</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QTextEdit" name="txtDescription"/>
      </item>
     </layout>
     <zorder>lblColor</zorder>
     <zorder>lblOpacity</zorder>
     <zorder>opacityPanel</zorder>
     <zorder>lblDescription</zorder>
     <zorder>txtDescription</zorder>
     <zorder>cmbColor</zorder>
     <zorder>textLabel2</zorder>
     <zorder>sliderOpacity</zorder>
     <zorder>textLabel1</zorder>
     <zorder>intNumLayers</zorder>
     <zorder>label_4</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>480</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>&amp;Create</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KisColorSpaceSelector</class>
   <extends>QWidget</extends>
   <header>widgets/kis_color_space_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>createButton</tabstop>
  <tabstop>txtName</tabstop>
  <tabstop>cmbPredefined</tabstop>
  <tabstop>doubleWidth</tabstop>
  <tabstop>doubleHeight</tabstop>
  <tabstop>cmbWidthUnit</tabstop>
  <tabstop>cmbHeightUnit</tabstop>
  <tabstop>bnPortrait</tabstop>
  <tabstop>bnLandscape</tabstop>
  <tabstop>doubleResolution</tabstop>
  <tabstop>bnScreenSize</tabstop>
  <tabstop>txtPredefinedName</tabstop>
  <tabstop>bnSaveAsPredefined</tabstop>
  <tabstop>clipPreview</tabstop>
  <tabstop>cmbColor</tabstop>
  <tabstop>intNumLayers</tabstop>
  <tabstop>sliderOpacity</tabstop>
  <tabstop>txtDescription</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
